backend/de.metas.business/src/main/java/de/metas/invoice/InvoiceCreditContext.java

```diff
@@ -43,4 +44,9 @@
 	 *         the credited quantities to be invoiced once more. Note that if the invoice's grand total is already partially allocated, this value will be override with <code>false</code>.
 	 */
 	boolean creditedInvoiceReinvoicable;
+
+	/**
+	 * @return if <code>true</code>, then the docActions <code>IDocument.ACTION_Reverse_Correct</code>, <code>IDocument.ACTION_ReActivate</code> and <code>IDocument.ACTION_Void</code> won't be available
```

Comment:
pls briefly describe what `fixedInvoice` means and not the consequence of what that.